teleport/datadog_checks/teleport/check.py

```diff
@@ -20,11 +20,18 @@ def __init__(self, name, init_config, instances):
 
     def check(self, _):
         try:
-            response = self.http.get("{}/healthz".format(self.diag_addr))
+            health_endpoint = f"{self.diag_addr}/healthz"
+            response = self.http.get(health_endpoint)
             response.raise_for_status()
-            self.service_check("health.up", self.OK)
+            self.count("health.up", 1, tags=["teleport_status:ok"])
         except Exception as e:
-            self.service_check("health.up", self.CRITICAL, message=str(e))
+            self.log.error(
+                "Cannot connect to Teleport HTTP diagnostic health endpoint '%s': %s.\nPlease make sure to enable Teleport's diagnostic HTTP endpoints.",  # noqa: E501
+                health_endpoint,
+                str(e),
+            )  # noqa: E501
+            self.count("health.up", 0, tags=["teleport_status:unreachable"])
```

Comment:
Can you add this new metric to the `metadata.csv`?

Reply:
Done